db/migrations/20210111183822_get_block_heights_for_transformed_diffs.sql

```diff
@@ -0,0 +1,14 @@
+-- +goose Up
+
+CREATE OR REPLACE FUNCTION api.get_block_heights_for_transformed_diffs()
```

Comment:
Pretty minor but maybe we should change the name of this function and the test file if we're only querying for one record